src/System.Net.Sockets/tests/PerformanceTests/SocketPerformanceAPMTests.cs

```diff
@@ -13,8 +13,6 @@ namespace System.Net.Sockets.Performance.Tests
     [Trait("Perf", "true")]
     public class SocketPerformanceAPMTests
     {
-        // NOTE: when enabled these tests will probably still fail on OS X due to #4009.
```

Comment:
Wouldn't we still hit this issue in OSX since we're opening >256 ports for the perf/stress tests?